frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/DisplayAttendees.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
don't use inline styles, create a hearings.scss in css module and add styles there, 

remember to import hearings.scss into index.scss